backend/ContextAlgorithm/ContextAlgorithm.py

```diff
@@ -0,0 +1,133 @@
+import requests
+import xml.etree.cElementTree as ET
+import string
+
+INPUT_LANG = 'es'
+OUTPUT_LANG = 'en'
+WORD = 'cama'
+WORD_DIFFICULTY = 882
+TITLE = 'SpanishWordFrequencies'
+PATH = "/home/antonio/Desktop/SE/FrecuencyList/" + TITLE + ".xml"
+MAX_LENGTH_SENTENCE = 15
+
+
+#Class sentence which contains:
+#   -the sentence(self.sentenceString)
+#   -words in the sentence(self.words)
+#   -values of each word(self.wordValues)
+#   -total difficulty of the sentence(self.difficulty)
+class Sentence:
+
+
+    def __init__(self, sentenceString, words):
```

Comment:
Having the variable type in the variable name is a good practice in python but Camel Case is not. Separate the different parts for the variable name with underscore